styles/wrap-guide.less

```diff
@@ -3,10 +3,22 @@
 atom-text-editor::shadow {
   .wrap-guide {
     height: 100%;
-    width: 1px;
+    width: 5px;
     z-index: 3;
     position: absolute;
     top: 0;
-    background-color: @syntax-wrap-guide-color;
+    background-color: transparent;
```

Comment:
Not sure. How should I go about checking if it does? What's the current workaround if it does? Using RGBA or HSLA with the alpha channel set to 0?